Dockerfile

```diff
@@ -2,37 +2,38 @@
 FROM ubuntu:latest
 MAINTAINER Nick Satterly <nick.satterly@theguardian.com>
 
-RUN apt-get update && apt-get install -y git wget build-essential python python-setuptools python-pip python-dev libffi-dev nginx
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    git \
+    libffi-dev \
+    nginx \
+    python \
+    python-dev \
+    python-pip \
+    python-setuptools \
+    wget
 
-RUN pip install --upgrade pip
+RUN pip install pip --upgrade
 RUN pip install uwsgi supervisor
 RUN pip install alerta-server alerta
 
 RUN wget -q -O - https://github.com/alerta/angular-alerta-webui/tarball/master | tar zxf -
-RUN mv alerta-angular-alerta-webui-*/app /app
+RUN mv alerta-angular-alerta-webui-*/app /app && rm -Rf /alerta-angular-alerta-webui-*
+
+ENV ALERTA_SVR_CONF_FILE /etc/alertad.conf
 
-ENV ALERTA_SVR_CONF_FILE /alertad.conf
 ENV BASE_URL /api
-ENV AUTH_REQUIRED False
```

Comment:
Do not include any settings that can be read from env var during runtime. See full list here https://github.com/guardian/alerta/blob/master/alerta/app/__init__.py